.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+// .devcontainer/devcontainer.json
+{
+	"name": "CUDA Python 3.12 Dev",
+  
+	"build": {
+	  "dockerfile": "DockerDevImage",  // relative to .devcontainer/
+	  "context": ".."              // repo root -> sees requirements.txt
+	},
+  
+	"runArgs": [
+	  "--gpus", "all"
```

Comment:
I suspect this is the thing that stops it - with a host-machine without GPU I just error out after building the devcontainer.